onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -1316,6 +1398,15 @@ TEST(ReductionOpTest, ReduceProd_int32) {
   test.Run();
 }
 
+#if !(defined USE_TENSORRT) && !(defined USE_TVM)
```

Comment:
instead of adding #if here, there's a EP exclusion list supported by test.Run(); example below line 1427
test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});  //TensorRT: axis must be 0